ProductsWebApi/ProductsWebApi.Tests/ValidatorTests/UserValidatorTest.cs

```diff
@@ -0,0 +1,64 @@
+﻿using AutoFixture;
+using Microsoft.AspNetCore.Identity;
+using ProductsWebApi.Models;
+using ProductsWebApi.Services;
+using ProductsWebApi.Validator;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using Xunit;
+
+namespace ProductsWebApi.Tests.ValidatorTests
+{
+    public class UserValidatorTest
+    {
+        private readonly Fixture _fixture;
+        private readonly UserValidator _userValidator;
+
+
+        public UserValidatorTest()
+        {
+            _fixture = new Fixture();
+            _userValidator = new UserValidator();
+        }
+
+        [Fact]
+        public void UserValidatorTests_WhenUserNameisEmpty_ShouldThrowValidationException()
+        {
+            var user = _fixture.Build<Users>().Create();
```

Comment:
This is weird. If the fixture generates the object, then the username would not be empty. Not sure, how the test case passes.